src/_locales/en/messages.json

```
@@ -99,6 +99,7 @@
     "error_gas_feed_exceeds": "Error occurred when invoking the smart contract.",
     "error_insufficient_balance": "Your allowance in this wallet is not sufficient to do that.",
     "error_unknown": "Unknown Error.",
+    "error_unknown_please": "Unknown Error; please ",
```

see https://react.i18next.com/latest/trans-component

```suggestion
    "error_unknown_please": "Unknown Error; please <again>try again</again>",
```

reference code:

https://github.com/DimensionDev/Maskbook/blob/c1cc8d28933337817490206cbcd8fa3f21846cdf/src/plugins/Gitcoin/DonateDialog.tsx#L176-L181

---

done